src/components/activities-test.js

```diff
@@ -240,6 +242,80 @@ describes.realWin('Activity Components', {}, (env) => {
         expect(original).to.be.instanceof(WebActivityPorts);
       });
     });
+
+    describe('openIframe', () => {
+      it('adds sut and publicationId', async () => {
+        const callMock = sandbox
+          .mock(activityPorts, 'openActivityIframePort_')
+          .expects('openActivityIframePort_')
+          .withExactArgs(
+            iframe,
+            `${url}?sut=${TOKEN}&publicationId=${publicationId}`,
+            {}
+          )
+          .once();
+        await activityPorts.openIframe(iframe, url, {});
+        callMock.verify();
+      });
+
+      it('does not add sut if it does not exist', async () => {
+        deps.storage = () => ({get: () => Promise.resolve(null)});
+        const callMock = sandbox
+          .mock(activityPorts, 'openActivityIframePort_')
+          .expects('openActivityIframePort_')
+          .withExactArgs(iframe, `${url}?publicationId=${publicationId}`, {})
+          .once();
+        await activityPorts.openIframe(iframe, url, {});
+        callMock.verify();
+      });
+
+      it('does not add publicationId if it does not exist', async () => {
+        deps.pageConfig = () => new PageConfig('', false);
+        const callMock = sandbox
+          .mock(activityPorts, 'openActivityIframePort_')
+          .expects('openActivityIframePort_')
+          .withExactArgs(iframe, `${url}?sut=${TOKEN}`, {})
+          .once();
+        await activityPorts.openIframe(iframe, url, {});
+        callMock.verify();
+      });
+
+      it('does not replace or duplicate already existing sut', async () => {
+        const callMock = sandbox
+          .mock(activityPorts, 'openActivityIframePort_')
+          .expects('openActivityIframePort_')
+          .withExactArgs(
+            iframe,
+            `${url}?sut=Original${TOKEN}&publicationId=${publicationId}`,
```

Comment:
There's no need for this to depend on the value of TOKEN.

Reply:
Done.